templates/data/kyma-integration-data.yaml

```diff
@@ -557,6 +563,9 @@ templates:
                     - "vm_job_labels_template"
               - jobConfig:
                   name: "pre-main-kyma-istio-reconcilation-k3d"
+                  annotations:
+                    owner: goat
+                    description: runs istio component tests
```

Comment:
I feel like this description doesn't helps us in understanding what is tested/done, since we already have jobs with exactly the same description and there must be something different for this job.